addons/bestja_detailed_reports/security/security.xml

```diff
@@ -0,0 +1,129 @@
+<openerp>
+    <data>
+        <record id="detailed_report_own_read_rule" model="ir.rule">
+            <field name="name">Managers / coordinators can see and create their own detailed_reports.</field>
+            <field name="model_id" ref="model_bestja_detailed_report"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                    ('project.manager', '=', user.id),
+                    ('project.organization.coordinator', '=', user.id),
+            ]</field>
+            <field name="perm_read" eval="True"/>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="True"/>
+            <field name="perm_unlink" eval="False"/>
+        </record>
+
+        <record id="detailed_report_own_modify_rule2" model="ir.rule">
+            <field name="name">Managers / coordinators can modify their own detailed reports</field>
+            <field name="model_id" ref="model_bestja_detailed_report"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                ('state', '=', 'accepted'),
```

Comment:
Change 
```
('state', '=', 'accepted'),
('project.organization.level', '=', 1),
```

to

```
'|',
   ('state', '!=', 'accepted'),
   ('project.organization.level', '=', 1),
```

And remove the nest rule (in other words merge this rule and the next rule together).